plugins/uefi-dbx/fu-uefi-dbx-plugin.c

```diff
@@ -35,7 +35,7 @@ fu_uefi_dbx_plugin_device_created(FuPlugin *plugin, FuDevice *device, GError **e
 	if (self->snapd_notifier != NULL) {
 		fu_uefi_dbx_device_set_snapd_notifier(FU_UEFI_DBX_DEVICE(device),
 						      self->snapd_notifier);
-	} else if (!inhibited && self->snapd_integration_supported && fu_snap_is_in_snap()) {
+	} else if (!inhibited && self->snapd_integration_supported) {
```

Comment:
would this make the intent clearer?
```suggestion
	} else if (!inhibited && self->snapd_integration_expected) {
```

Reply:
Hm, I think supported is fine? It indicates that we have already done the check to make sure that the support is there.